one/src/daemon.rs

```diff
@@ -323,11 +323,33 @@ fn spawn_database_optimizer(
     mut shutdown: tokio::sync::broadcast::Receiver<()>,
 ) -> tokio::task::JoinHandle<()> {
     tokio::spawn(async move {
+        // Recommended practice is that applications with long-lived database connections should run "PRAGMA optimize=0x10002"
+        // when the database connection first opens, then run "PRAGMA optimize" again at periodic intervals - perhaps once per day
+        // https://www.sqlite.org/pragma.html#pragma_optimize
+        match sqlite_pool.run_statement("PRAGMA optimize=0x10002").await {
```

Comment:
How is this statement different from `sqlite_pool.optimize()`? Do we need to run this statement once per day as well? The comment implies we do but the code does not.

Reply:
I added some more comments about it and refactored when/how it's called. It's supposed to be called that way once at startup (`Run ANALYZE on tables that might benefit. On by default`) so it doesn't seem strictly necessary to include but I included it jic.